dashboard/config/linux/bits/base.yml

```diff
@@ -131,7 +131,9 @@ config:
  - KCOV_INSTRUMENT_ALL: [-arm, -s390, -nokcov]
  # Doesn't boot with KCOV_ENABLE_COMPARISONS on Cuttlefish.
  - KCOV_ENABLE_COMPARISONS: [-arm, -s390, -nokcov, -cuttlefish]
- - KCOV_SELFTEST: [linux-next, -baseline, -arm, -s390, -nokcov]
+ # As of Sep 2024, the test does not pass (at least) on gcc-built kernels.
+ # TODO: re-enable the option once the problem is addressed.
```

Comment:
The report link would be useful here as well.

Reply:
PTAL